docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,457 @@
+
+**Fixed** SQL Server connection leak issue
+
+**Updated** Kubernetes events input binding with new payloads
+
+### Java SDK
+
+**Added** multi pub/sub support
+
+**Updated** k8s annotations, CLI options and daprd arguments
+
+**Updated** API to support HTTP extensions
+
+**Removed** retry options from state
+
+**Fixed** gRPC manager not shutting down
+
+### .NET SDK
+
+**Added** multi pub/sub support
+
+**Updated** actor usage docs (Thanks @tomkerkhove)
+
+**Updated** proto package with Any type (Thanks @akkie)
+
+**Fixed** null reference exception when no model binder provided (Thanks @altinoren)
+
+**Added** Bulk Get API
+
+**Updated** k8s annotations, CLI options and daprd arguments
+
+**Removed** retry options from state
+
+**Added** api token injection
+
+### Go SDK
+
+**Added** multi pub/sub support
+
+**Removed** retry options from state
+
+**Updated** k8s annotations, CLI options and daprd arguments
+
+**Added** server side callback wrapper
+
+**Added** Bulk Get API
+
+**Added** state transactions API
+
+### Python SDK
+
+**Added** multi pub/sub support
+
+**Updated** k8s annotations, CLI options and daprd arguments
+
+**Added** wrapper methods for state store APIs
+
+**Added** wrapper layers for gRPC
+
+**Added** server side wrapper for app callback
+
+**Added** actor service via fastapi
+
+**Added** wrapper methods for secrets API
+
+**Added** wrapper methods for pub/sub API
+
+**Added** wrapper methods for bindings API
+
+**Added** support for api token auth
+
+### Rust SDK
+
+**Added** multi pub/sub support
+
+**Updated** k8s annotations, CLI options and daprd arguments
+
+### Javascript SDK
+
+**Added** multi pub/sub support
+
+**Updated** k8s annotations, CLI options and daprd arguments
+
+### Docs
+
+**Added** Kubernetes production deployment and upgrade guidelines
+
+**Updated** multi pub/sub parts
+
+**Updated** k8s annotations, CLI and daprd arguments
+
+**Fixed** broken KEDA link (Thanks @tomkerkhove)
+
+**Updated** with new IntelliJ instructions for multiple dapr runs
+
+**Updated** API reference with state transactions API
+
+**Updated** API reference with state bulk get API
+
+**Updated** Kubernetes events input binding
+
+**Updated** troubleshoot section for configuring multiple replicas to the Kubernetes sidecar injector
+
+**Updated** state API reference to remove retry options
+
+**Updated** links to reference to new quickstarts repo
+
+## Upgrading to Dapr 0.10.0
+
+If you're upgrading from an older version of Dapr to 0.10.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have. Note that this will remove the default .dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo.:
```

Comment:
Would it be worth pointing out where the `.dapr` dir is located? In case I want to do something before removing

Reply:
Added